netlify/next-runtime
# chore: delete e2e deploys in afterAll

### Summary

For our e2e tests we already run an `afterAll` call for every test. There we call `.destroy()`.

I've extended the NextDeployInstance class to override the base `destroy()` method. It now deletes the deploy that the class 
created in the `setup()` method.

It does this by calling the `netlify api` with `deleteDeploy`

### Test plan

1. See if tests that should pass still pass
1. Check if deploys are actually deleted

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

Fixes https://github.com/netlify/pod-ecosystem-frameworks/issues/423
